Source/WebInspectorUI/UserInterface/Views/Main.css

```diff
@@ -165,26 +165,46 @@ input[type=range]::-webkit-slider-runnable-track {
 #navigation-sidebar {
     width: 300px;
     height: 100%;
+    grid-area: navigation-sidebar;
 }
 
 body.docked:is(.right, .left) #navigation-sidebar.collapsed > .resizer {
     pointer-events: none;
 }
 
 #content {
-    display: flex;
+    display: grid;
     height: 100%; /* This reduces paint areas when typing in the console. http://webkit.org/b/145324 */
-    flex: 1;
+    grid-template-columns: auto 1fr auto;
+    grid-template-areas: "navigation-sidebar tab-browser details-sidebar";
```

Comment:
nit: we could move this into a `body:not(.narrow) #content` like we do below so that there's never more than one `grid-template-columns`/`grid-template-areas` even attempting to apply to the element at the same time